src/mlpack/tests/CMakeLists.txt

```diff
@@ -122,7 +122,11 @@ add_executable(mlpack_test
   ub_tree_test.cpp
   union_find_test.cpp
   vantage_point_tree_test.cpp
-  main_tests/pca_test.cpp
+  main_tests/decision_tree_test.cpp
+# main_tests/pca_test.cpp
+# main_tests/preprocess_binarize_test.cpp
+# main_tests/preprocess_imputer_test.cpp
+# main_tests/preprocess_split_test.cpp
```

Comment:
Now that #1172 is merged, can you update your branch and uncomment these tests?  Thanks!